sdk-cpp/tests/config_tests.cpp

```diff
@@ -47,6 +47,20 @@ TEST_F(ConfigTests, IoTConnectionString)
     ASSERT_EQ(value, std::string{expectedValue});
 }
 
+TEST_F(ConfigTests, DOCacheHost)
+{
+    // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Not credentials (false positive)")]
```

Comment:
Can probably remove the cred scan line here, and also add existing cred scan lines to the supression json file